packages/react-composites/src/composites/CallComposite/components/buttons/EndCall.tsx

```diff
@@ -28,7 +28,30 @@ export const EndCall = (props: {
   const localeStrings = useLocale().strings;
 
   /* @conditional-compile-remove(end-call-options) */
-  const { onHangUp, enableEndCallMenu } = hangUpButtonProps;
+  const endCallDiaglogLabels = useMemo(
+    () => ({
+      confirmButtonLabel: localeStrings.call.endCallConfirmButtonLabel,
+      heading: localeStrings.call.leaveConfirmDialogTitle,
```

Comment:
This should use different strings than leaveDialogLabels I think?

Reply:
Nice catch! Fixed!